platforms/software/maven/src/integTest/groovy/org/gradle/api/publish/maven/MavenPublishPomCustomizationIntegTest.groovy

```diff
@@ -113,6 +113,11 @@ class MavenPublishPomCustomizationIntegTest extends AbstractMavenPublishIntegTes
                                     version = "42"
                                     message = "the answer to life, the universe and everything"
                                 }
+                                repository {
+                                    id = "internalMaven"
+                                    name = "Our internal maven repo"
+                                    url = "https://internal.maven.repo.com"
```

Comment:
Can we use a guaranteed safe URL here? e.g., example.com (for documentation purposes) or example.invalid (not resolvable)